module_a/src/main/java/org/sonarqube/A.java

```diff
@@ -3,10 +3,12 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: The return value of "length" must be used. ([squid:S2201](http://localhost:9000/coding_rules#rule_key=squid%3AS2201))

[See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_33&issues=AWB9UVKNR10kk73Q_CC0&open=AWB9UVKNR10kk73Q_CC0)

[ISSUE_KEY]: AWB9UVKNR10kk73Q_CC0

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED